libreflexxestype2/CMakeLists.txt

```diff
@@ -29,24 +30,31 @@ if(CMAKE_DPKG)
 endif()
 
 if(SUPPORTED MATCHES "true")
- 
-  add_custom_target(
-    build_reflexxes ALL
+  # Set location of build output
+  set(libreflexxes_LOCATION ${CMAKE_CURRENT_SOURCE_DIR}/Linux/x64/release/lib/shared/libReflexxesTypeII.so)
+
+  # Build Reflexxes library with a Make file
+  add_custom_command(
+    OUTPUT ${libreflexxes_LOCATION}
     COMMAND make -C "${CMAKE_CURRENT_SOURCE_DIR}/Linux" clean64 all64 WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-    # copy libs, set-up soname chain
-    COMMAND ${CMAKE_COMMAND} -E copy_if_different ${PROJECT_SOURCE_DIR}/Linux/x64/release/lib/shared/libReflexxesTypeII.so ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_LIB_DESTINATION}
-    COMMAND ${CMAKE_COMMAND} -E create_symlink libReflexxesTypeII.so ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_LIB_DESTINATION}/libreflexxes.so
   )
+  add_library(reflexxes ${libreflexxes_LOCATION})
```

Comment:
OK, usage of `${libreflexxes_LOCATION}` seems like the important thing I was missing. Nice